### PR TITLE
Uploading files for the project Awesome-Books-Classes

### PR DESCRIPTION
**Pull request by siddghosh108**
PR Description

Files created:
index.html
styles.css
index.js
README.md
mit.md

Please provide your review
Thanks

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @siddghosh108,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### - @siddghosh108 has a second profile, @sghosh108. Kindly ensure only the github account you registered with Microverse is what you use for projects. Kindly redo the project from scratch using the Microverse account. 

### - The other coding partner did not contribute to this project at all. Kindly ensure all coding partners contribute to this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @siddghosh108,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly take into account that you need to implement the changes requested previously before re-submitting for reviews. Only your Microverse account should commit to this project and your coding partner needs to commit to this project as well. These entail that you start the project from scratch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+const objectOne = new Bookstore();
+
+function removeBook(index) {
+  objectOne.bookLibrary = objectOne.bookLibrary.filter((book, bookIndex) => bookIndex !== index);
+  localStorage.setItem('books', JSON.stringify(objectOne.bookLibrary));
+
+  objectOne.createLibrary();
+}
```
Comment (rloterh) [inline js/index.js]: - [ ] In this activity we require that all objects, array or similar, are placed inside of the class. Let's therefore ensure to refactor the `removeBook` function into the `Bookstore`class created.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Zerihun & Siddhartha, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🚦 __🏃 🏡 🏃‍♂️ 🏃‍♀️ 🏚️ 🌴

## Highlights 👍  
- Good job so far on refactoring your code!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Kindly include details of the other co-contributor(s) in your ReadMe.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @siddghosh108 @znation2010 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Setup linters correctly ✔️ 
- Great use of JavaScript class ✔️ 
- Application works properly ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**